File: archive/p/perl/fileIO.pl
#!/usr/bin/env perl

open(my $writing, ">output.txt") || die "File could not be written.\nError: $!";

print $writing "Some arbitrary data.\n";

close($writing) || die "The file could not be closed on write.\nError: $!";

open(my $reading, "<output.txt") || die "File could not be readed.\nError: $!";

while (!eof($reading)) {
	print <$reading>;
}

close($reading) || die "The file could not be closed on reading.\nError: $!";

exit(0);

File: archive/l/lua/file-io.lua
function writer()
	file_to_be_written = io.open("output.txt","w+")
	io.output(file_to_be_written)
	io.write("text to be written into output.txt")
	io.close(file_to_be_written)
end


function reader()
	file_to_be_read = io.open("output.txt","r")
	io.input(file_to_be_read)
	print(io.read())
	io.close(file_to_be_read)
end

writer()
reader()
